### ISSUE TITLE
6.2 Beta 4 release

### ISSUE THREAD

**Issue by supermansrf** [ActiveForums/ActiveForums#281]
Few small corrections.  I would suggest installing the PA, then doing a repair install.  I have to each time I install a new version. Trying to determine if this is my server only or all of them.  It appears the IIS temp folder isn't clearing properly after install.  

Included pull #280 to correct quick jump menu on responsive with long forum names.  Moved module branding icon to proper folder and adjusted manifest to match.

**Comment by supermansrf** [comment]
I can confirm that the issue @plumifrons reported does exist.

![image](https://cloud.githubusercontent.com/assets/6083121/7803621/bc1a1a9e-0322-11e5-8959-a7ef6e893b63.png)

**Comment by supermansrf** [comment]
So I can confirm that it appears to be all images.  If I insert an image via attach, then insert inline, the photo appears to be growing to max-width.  We have run across this before, but I honestly don't remember how we fixed this before.

**Comment by supermansrf** [comment]
The problem is with module.css line 58 

When I uncheck width=100% it goes to normal.  I don't know what repercussions exist from commenting this out.  Going to let the experts answer

**Comment by supermansrf** [comment]
If you comment out module.css line 59, change it to:
/*width: 100%*/

The problem goes away.  It appears from looking at the module.css file, there are future plans to remove some of the css down the road.  I have done this temporarily, just waiting until i hear a proper answer to make the change.

**Comment by supermansrf** [comment]
It removed my stars, should be / * width: 100% * / without the extra space between * and /

**Comment by Plumifrons** [comment]
I don’t see the 24h time notation. The settings are HH:mm tt
![2015-05-26 20_10_43-my website - af - default group - sample forum 1 - internet explorer](https://cloud.githubusercontent.com/assets/5980007/7819665/84850186-03e3-11e5-9a80-b7181402c857.png)

**Comment by Plumifrons** [comment]
Saving ActiveForums settings gives errors in the event log.


ModuleId:410
ModuleDefId:126
FriendlyName:Active Forums
ModuleControlSource:DesktopModules/ActiveForums/ActiveForums.ascx
AbsoluteURL:/Default.aspx
DefaultDataProvider:DotNetNuke.Data.SqlDataProvider, DotNetNuke
ExceptionGUID:fd6cda3c-3cf8-46ea-a4eb-70f28ba0e9b9
AssemblyVersion:7.4.1
PortalId:0
UserId:1
TabId:87
RawUrl:/AF/ctl/Module/ModuleId/410?ReturnURL=/AF&popUp=true
Referrer:http://www.dnnhost.nl/AF/ctl/Module/ModuleId/410?ReturnURL=/AF&popUp=true
UserAgent:Mozilla/5.0 (Windows NT 6.3; WOW64; Trident/7.0; rv:11.0) like Gecko
ExceptionHash:mYpFcvezC3Qnr8iJPLjuVA==
Message:The stored procedure 'dbo.activeforums_Search_ManageFullText' doesn't exist.
StackTrace:
InnerMessage:The stored procedure 'dbo.activeforums_Search_ManageFullText' doesn't exist.
InnerStackTrace:
   at System.Data.SqlClient.SqlCommand.DeriveParameters()
   at System.Data.SqlClient.SqlCommandBuilder.DeriveParameters(SqlCommand command)
   at Microsoft.ApplicationBlocks.Data.SqlHelperParameterCache.DiscoverSpParameterSet(SqlConnection connection, String spName, Boolean includeReturnValueParameter, Object[] parameterValues)
   at Microsoft.ApplicationBlocks.Data.SqlHelperParameterCache.GetSpParameterSetInternal(SqlConnection connection, String spName, Boolean includeReturnValueParameter)
   at Microsoft.ApplicationBlocks.Data.SqlHelperParameterCache.GetSpParameterSet(String connectionString, String spName, Boolean includeReturnValueParameter)
   at Microsoft.ApplicationBlocks.Data.SqlHelper.ExecuteScalar(String connectionString, String spName, Object[] parameterValues)
   at DotNetNuke.Modules.ActiveForums.SqlDataProvider.Search_ManageFullText(Boolean enabled)
   at DotNetNuke.Modules.ActiveForums.Controls.ForumSettings.UpdateSettings()
Source:
FileName:
FileLineNumber:0
FileColumnNumber:0
Method:



AbsoluteURL:/Default.aspx
DefaultDataProvider:DotNetNuke.Data.SqlDataProvider, DotNetNuke
ExceptionGUID:904cd7bf-d830-481c-a46f-0588ddad9dce
AssemblyVersion:7.4.1
PortalId:0
UserId:1
TabId:87
RawUrl:/AF/ctl/Module/ModuleId/410?ReturnURL=/AF&popUp=true
Referrer:http://www.dnnhost.nl/AF/ctl/Module/ModuleId/410?ReturnURL=/AF&popUp=true
UserAgent:Mozilla/5.0 (Windows NT 6.3; WOW64; Trident/7.0; rv:11.0) like Gecko
ExceptionHash:Cg/j+haGfYZieVjDcsPSCA==
Message:The Controls collection cannot be modified because the control contains code blocks (i.e. <% ... %>).
StackTrace:
InnerMessage:The Controls collection cannot be modified because the control contains code blocks (i.e. <% ... %>).
InnerStackTrace:
   at System.Web.UI.ControlCollection.Add(Control child)
   at DotNetNuke.Services.Exceptions.Exceptions.ProcessModuleLoadException(String FriendlyMessage, Control ctrl, Exception exc, Boolean DisplayErrorMessage)
Source:
FileName:
FileLineNumber:0
FileColumnNumber:0
Method:

**Comment by Plumifrons** [comment]
The icons in the Telerik editor are not placed correct when editing a topic or reply. In the normal html module they are okay.
![2015-05-27 17_06_42-my website _ af - internet explorer](https://cloud.githubusercontent.com/assets/5980007/7845605/465f1786-04b8-11e5-992c-35045742d77f.png)

**Comment by laralmix** [comment]
"When I uncheck width=100% it goes to normal. I don't know what repercussions exist from commenting this out. Going to let the experts answer"
I added this line because "max-width:100%" does not work for images in this context (though it should work in theory). On the other hand this option affects "div". It seems to be just a bug (I checked it in FireFox and Chrome). The problem with small icons like the smiles can be fixed by adding the "width" attribute. Also it is possible to add a certain class to attached (big) images with width=100% which will not affect other images.
